Remotion/ObjectBinding/Web/UI/Controls/BocBooleanValueImplementation/Rendering/BocCheckBoxRenderer.cs

```diff
@@ -222,45 +227,52 @@ private void RegisterStartupScriptIfNeeded (BocCheckBoxRenderingContext renderin
 
       string startupScript = string.Format (
           "BocCheckBox.InitializeGlobals ('{0}', '{1}');",
-          renderingContext.Control.DefaultTrueDescription,
-          renderingContext.Control.DefaultFalseDescription);
+          ScriptUtility.EscapeClientScript (renderingContext.Control.DefaultTrueDescription.ToString (WebStringEncoding.HtmlWithTransformedLineBreaks)),
+          ScriptUtility.EscapeClientScript (renderingContext.Control.DefaultFalseDescription.ToString (WebStringEncoding.HtmlWithTransformedLineBreaks)));
       renderingContext.Control.Page.ClientScript.RegisterStartupScriptBlock (renderingContext.Control, typeof (BocCheckBoxRenderer), s_startUpScriptKey, startupScript);
     }
 
-    private void PrepareImage (BocCheckBoxRenderingContext renderingContext, Image imageControl, string description)
+    private void PrepareImage (BocCheckBoxRenderingContext renderingContext, Image imageControl, WebString description)
     {
       var imageUrl = ResourceUrlFactory.CreateThemedResourceUrl (
           typeof (HtmlHeadAppenderExtensions),
           ResourceType.Image,
           renderingContext.Control.Value.Value ? c_trueIcon : c_falseIcon);
 
       imageControl.ImageUrl = imageUrl.GetUrl();
-      imageControl.AlternateText = description ?? string.Empty;
+      imageControl.AlternateText = description.GetValue();
       imageControl.GenerateEmptyAlternateText = true;
     }
 
-    private void PrepareLabel (BocCheckBoxRenderingContext renderingContext, string description, Label labelControl)
+    private void PrepareLabel (BocCheckBoxRenderingContext renderingContext, WebString description, Label labelControl)
     {
       labelControl.CssClass = "description";
-      labelControl.Text = description;
+      labelControl.Text = description.ToString (WebStringEncoding.HtmlWithTransformedLineBreaks);
       labelControl.Width = Unit.Empty;
       labelControl.Height = Unit.Empty;
       labelControl.ApplyStyle (renderingContext.Control.LabelStyle);
     }
 
-    private string GetDescription (BocCheckBoxRenderingContext renderingContext)
+    private WebString GetDescription (BocCheckBoxRenderingContext renderingContext)
     {
-      string trueDescription = null;
-      string falseDescription = null;
+      WebString? trueDescription = null;
+      WebString? falseDescription = null;
```

Comment:
Question about stylistic approach: why not set this to WebString.Empty? I guess either is fine but using WebString.Empty seems like it's less "noise".

Reply:
Makes sense, I'll change it